### PR TITLE
fix: polish the print of dice action

### PR DESCRIPTION
**Pull request by littlejiancc**
**What type of this PR**
/kind polish

**What this PR does / why we need it:**
fix: polish the print of dice action

**Which issue(s) this PR fixes:**
https://erda.cloud/erda/dop/projects/387/issues/bug?id=204242&issueFilter__urlQuery=eyJpdGVyYXRpb25JRHMiOls0MzFdLCJzdGF0ZUJlbG9uZ3MiOlsiT1BFTiIsIldPUktJTkciLCJXT05URklYIiwiUkVPUEVOIiwiUkVTT0xWRUQiXSwiYXNzaWduZWVJRHMiOlsiMTAwMTI2MSJdfQ%3D%3D&issueTable__urlQuery=eyJwYWdlTm8iOjEsICJwYWdlU2l6ZSI6MTB9&issueViewGroup__urlQuery=eyJ2YWx1ZSI6InRhYmxlIiwiY2hpbGRyZW5WYWx1ZSI6eyJrYW5iYW4iOiJkZWFkbGluZSJ9fQ%3D%3D&iterationID=431&type=BUG

reviewer: @sfwn

### CODE REVIEW

File: actions/dice/1.0/internal/dice/client.go [hunk 1]
```diff
@@ -46,6 +48,19 @@ type deployRequest struct {
 	Extra        map[string]interface{} `json:"extra, omitempty"`
 }
 
+func (req *deployRequest) print() {
+	log.AddNewLine(1)
+	logrus.Infof("request deploy body: ")
+	logrus.Infof(" clusterName: %s", req.ClusterName)
```
Comment (sfwn) [inline actions/dice/1.0/internal/dice/client.go:54]: I think `C` should be uppercase, the same with others.

File: actions/dice/1.0/internal/dice/client.go [hunk 2]
```diff
@@ -79,7 +94,7 @@ func (d *dice) Deploy(deployReq *deployRequest, conf *conf) (*DeployResult, erro
 	}, 5, time.Second*3)
 
 	if err != nil {
-		logrus.Errorf("deploy to dice failed! response err:%v.", err)
+		logrus.Errorf("deploy to dice failed! response err: %v.", err)
```
Comment (sfwn) [inline actions/dice/1.0/internal/dice/client.go:97]: no need end dot.

File: actions/dice/1.0/internal/dice/client.go [hunk 3]
```diff
@@ -103,23 +118,55 @@ type R struct {
...
+	}
+	if len(r.Data.MoudleErrMsg) > 0 {
+		for k, v := range r.Data.MoudleErrMsg {
+			if v != "" {
+				logrus.Infof(" %s: %s", k, v)
+			}
+		}
+	}
+	if r.Data.Runtime != nil {
+		logrus.Infof(" runtime: %+v", r.Data.Runtime)
```
Comment (sfwn) [inline actions/dice/1.0/internal/dice/client.go:138]: Do not use %+v.
Print necessary fields only. Or use json format.

File: actions/dice/1.0/internal/dice/client.go [hunk 4]
```diff
@@ -103,23 +118,55 @@ type R struct {
...
+	}
+
+	logrus.Infof(" err: %+v", r.Err)
```
Comment (sfwn) [inline actions/dice/1.0/internal/dice/client.go:141]: Err may be nil.

**Review by sfwn — CHANGES_REQUESTED** [review]

File: actions/dice/1.0/internal/dice/client.go (hunk 3 above)
Comment (littlejiancc) [reply actions/dice/1.0/internal/dice/client.go:138]: the Runtime is interface{}

File: actions/dice/1.0/internal/dice/client.go (hunk 1 above)
Comment (littlejiancc) [reply actions/dice/1.0/internal/dice/client.go:54]: done

File: actions/dice/1.0/internal/dice/client.go (hunk 2 above)
Comment (littlejiancc) [reply actions/dice/1.0/internal/dice/client.go:97]: done

File: actions/dice/1.0/internal/dice/client.go (hunk 1 above)
Comment (littlejiancc) [reply actions/dice/1.0/internal/dice/client.go:54]: all update to lowercase

File: actions/dice/1.0/internal/dice/client.go (hunk 4 above)
Comment (littlejiancc) [reply actions/dice/1.0/internal/dice/client.go:141]: done

File: actions/dice/1.0/internal/dice/deploy.go [hunk 5]
```diff
@@ -51,6 +52,9 @@ func Run() error {
 			logrus.Warningf("Report runtimeID to ci failed.")
 		}
 	}
+	log.AddNewLine(1)
+	logrus.Infof("msg=\"deploy starting... ##to_link:applicationId:%d,runtimeId:%d,deploymentId:%d\"",
```
[the remark this answers is not in the document]
Comment (littlejiancc) [reply actions/dice/1.0/internal/dice/deploy.go:56]: done

**Review by sfwn — APPROVED** [review]

**Pull request merged**